docs/commands.md

```
@@ -92,7 +94,43 @@ camel-kit init my-integration --ai claude --source-platform biztalk
+# Overwrite an existing project
+camel-kit init my-integration --ai claude --force
+
+# Check version
+camel-kit --version
+```
+
+**Prerequisite check:**
+
+On startup, `init` checks for required tools and reports their status:
+
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)